build

```diff
@@ -411,6 +422,7 @@ case "$gitstatus_kernel" in
           armv6l)         docker_image=arm32v6/alpine:3.11.6;;
           armv7l)         docker_image=arm32v7/alpine:3.11.6;;
           aarch64)        docker_image=arm64v8/alpine:3.11.6;;
+          ppc64le)        docker_image=ppc64le/alpine:3.11.6;;
```

Comment:
Do you know if alpine has plans to support riscv64? I really like building on alpine because it's so small and has musl (for static linking) out of the box.

(This is not a blocker for your PR. Supporting local builds for riscv64 is already awesome!)

Reply:
Unfortunately there are no official Docker images for riscv64 yet.